data/hails.txt

```diff
@@ -3131,7 +3226,6 @@ phrase "hostile free worlds"
 		"."
 		"!"
 
-
```

Comment:
Do not make whitespace changes like these, please.

Reply:
As I commented in a previous one, it was unintentional but afaik it was something I added when I tried to replace something that my copy didnt originally have, for whatever reason, the test dummy lines.

Reply:
So add it back in. :slightly_smiling_face:

Reply:
no
```
	word
		"."
		"!"

phrase "hostile free worlds"
	word
		"We "
	word
```
```
	word
		"."
		"!"

# Hails from the robotic Syndicate test dummy ships
phrase "test dummy"
	word
```
There is nothing wrong with the spacing.